projects/ngx-launcher/src/lib/import-app/project-progress-importapp-nextstep/project-progress-importapp-nextstep.component.ts

```diff
@@ -57,12 +60,22 @@ export class ProjectProgressImportappNextstepComponent implements OnChanges, OnD
         status.hyperText = data.location;
       }
     }
-  };
+  }
 
   ngOnDestroy() {
     this.closeConnections();
   }
 
+  addQuery() {
```

Comment:
I think there is a better way to pass a parameter in a route

Reply:
Do you have anything specific in mind or is it just a matter of the json string?

Reply:
why not just use a param parameter? but anyway it's out of scope for this PR